quickOrder/database_abstraction.py

```diff
@@ -48,4 +48,18 @@ def updateOrderName(order_id, item_id):
 def getUserEmail(user_id):
     return User.objects.get(id = user_id).email
 
-    
\ No newline at end of file
+
+def addStatusQuantity():
+    quantity = []
+    inventory = getAllInventoryObjects()
+    for item in inventory:
+        orders = Order.objects.all().filter(item_id = item.id)
+        pendingCount = 0
+        approvedCount = 0
+        for order in orders:
+            if order.status == "Pending Approval":
+                pendingCount += order.requested_quantity
+            if order.status == "Approved":
+                approvedCount += order.requested_quantity
+        quantity.append([pendingCount, approvedCount])
```

Comment:
Can we think of doing this loop via queries (table inner join) itself.

Don't you need to add item id in quantity array?